Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json

```diff
@@ -4,128 +4,394 @@
+                    "text": "Yukarıda: %11,25'lik bir düşüşle orta kuvvette bir düşüş (kırmızı) ve bir yükseliş (yeşil) sapma görüyorsunuz.",
+                    "updated": 1647002286049
+                }
+            ]
         },
-		{
+        {
             "style": "Png",
             "text": "PNGs/Foundations/Docs/indicators/Divergences-RSI-2.png"
         },
-		{
+        {
             "style": "Text",
-            "text": "Above: a hidden bullish (dashed green) and a bullish (green) divergence with strong strength and 70% increase."
+            "text": "Above: a hidden bullish (dashed green) and a bullish (green) divergence with strong strength and 70% increase.",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Выше: скрытая бычья (зеленый пунктир) и бычья (зеленый) дивергенция с сильной силой и 70-процентным ростом.",
+                    "updated": 1645608520965
+                },
+                {
+                    "language": "TR",
+                    "text": "Yukarıda: güçlü bir güç ve %70 artış ile gizli bir yükseliş (kesikli yeşil) ve bir yükseliş (yeşil) ayrışması görüyorsunuz.",
+                    "updated": 1647002280026
+                }
+            ]
         },
-		{
+        {
             "style": "Text",
-            "text": "The length of the divergence finder, the pivot length, as well as the threshold for strong and weak divergences can be changed by locating and opening the Javascript Code under Data Building Procedure -> Procedure Loop under \u2019RSI Divergence\u2019 Procedure Loop."
-        },	
+            "text": "The length of the divergence finder, the pivot length, as well as the threshold for strong and weak divergences can be changed by locating and opening the Javascript Code under Data Building Procedure -> Procedure Loop under ’RSI Divergence’ Procedure Loop.",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Длина детектора дивергенции, длина разворота, а также порог для сильных и слабых дивергенций могут быть изменены путем нахождения и открытия Javascript Code в разделе Data Building Procedure -> Procedure Loop в разделе'RSI Divergence'Procedure Loop.",
+                    "updated": 1645608623722
+                },
+                {
+                    "language": "TR",
+                    "text": "Farklılıklar bulucusunun uzunluğu, pivot uzunluğu ve ayrıca güçlü ve zayıf sapmalar için eşik, Javascript Kodunun ( Javascript Code ) 'RSI Sapma'Prosedür Döngüsü altındaki Veri Oluşturma Prosedürü ( Data Building Procedure ) -> Prosedür Döngüsü ( Procedure Loop ) altında bulunması ve açılmasıyla değiştirilebilir.",
+                    "updated": 1647002818095
+                }
+            ]
+        },
         {
             "style": "Png",
             "text": "PNGs/Foundations/Docs/indicators/Divergences-RSI-params-1.png"
-        },		
```

Comment:
Is this part still needed when down below there comes:
style
text 
translations?

Reply:
not sure what you mean by this?

Reply:
you delete a small part here. Above there is already:
style
text
 that wont get deleted. but down below the part where you delete something there got added :
style
text
translations
 and Iam unsure if the first part is a duplicate or if this is just the syntax.

Reply:
hmm strange, that file was duplicated altogether so i removed the incorrect one and added one with the filename divergences-data-mine-002-rsi-divergence.json. Maybe git is being weird about the filename case on the new file.

Reply:
So now everything should be correct?

Reply:
yeah should be good there now I think!